src/coreclr/jit/valuenum.cpp

```diff
@@ -1672,6 +1678,33 @@ ValueNumStore::Chunk::Chunk(CompAllocator alloc, ValueNum* pNextBaseVN, var_type
                     // Since this value is always the same, we represent it as a static.
                     m_defs = &s_specialRefConsts[0];
                     break; // Nothing to do.
+
+#if defined(FEATURE_SIMD)
+                case TYP_SIMD8:
+                {
+                    m_defs = new (alloc) Alloc<TYP_SIMD8>::Type[ChunkSize];
+                    break;
+                }
```

Comment:
This is new code under an ifdef and we have many places where there is a mix of `braces` and `no-braces` depending on the needs of the logic and if its new or not.

It's not worth churning the existing code in an unrelated PR.

Reply:
I think it's unfortunate to have the same function have two styles, but so be it :).

Reply:
Yes, and I'd like to fix everything to use braces more generally. Not having braces is up there with `if (someInt)` and other problematic patterns in C/C++ as a source of bugs.

Unfortunately, the existing pattern is a mix. Sometimes due to cases like this where new code has added braces where existing code didn't but also simply because some C/C++ code requires braces and other code does not and so its naturally crept in.